src/languages/en.js

```diff
@@ -476,12 +479,19 @@ export default {
     setPasswordPage: {
         enterPassword: 'Enter a password',
         setPassword: 'Set password',
+        confirmPassword: 'Confirm Password',
+        confirmPasswordInvalidMessage: 'Please make sure the confirm password is match',
         newPasswordPrompt: 'Your password must have at least 8 characters, 1 capital letter, 1 lowercase letter, and 1 number.',
-        passwordFormTitle: 'Welcome back to the New Expensify! Please set your password.',
+        passwordFormTitle: 'Please set a password below to sign into your account',
         passwordNotSet: 'We were unable to set your new password. We have sent you a new password link to try again.',
         setPasswordLinkInvalid: 'This set password link is invalid or has expired. A new one is waiting for you in your email inbox!',
         verifyingAccount: 'Verifying account',
     },
+    welcomeProfilePage: {
+        formTitle: 'Welcome to Expensify! Tell us a little bit about yourself before getting started.',
+        requestTimeoutErrorMessage: 'Request taking too long than expected, some network request could be failed. Please try again.',
```

Comment:
Was this copy requested from marketing? It doesn't really sound like it and the grammar isn't correct (eg. "too long than expected" and "could be failed" doesn't make sense). I would suggest getting copy from marketing for this so that the voice is consistent.

Reply:
Yes, I have gathered all message [here](https://github.com/Expensify/App/pull/7930#issuecomment-1053879304). Need suggestion by internal or marketing people.